### PR TITLE
Tabu tables learn 'spread' and 'to' syntax

### PR DESCRIPTION
**Pull request by s-wallace**
Tabu (and LongTabu) tables support an unusual Latex syntax for additional width control, these are now accessible to users of the Tabu/LongTabu classes through additional __init__ arguments

\begin{tabu} spread 0pt {...}

and 
\begin{tabu} to 5in {...}

### CODE REVIEW

File: pylatex/table.py [hunk 1]
```diff
@@ -378,6 +378,77 @@ class Tabu(Tabular):
...
+
+
+
+        super().__init__(table_spec, data, pos,
+                         row_height=row_height, col_space=col_space,
+                         width=width, booktabs=booktabs, **kwargs)
+
+        self._preamble = ""
+        if spread:
+            self._preamble = "spread " + spread
```
Comment (JelteF) [inline pylatex/table.py]: Isn't it possible to just prepend these two new options to `self.data` here? That would make the dumps changes below with their hardcoded lengths not necessary anymore.

**Comment by JelteF** [comment]
Good catch for the missing functionality! Let me know if my idea works.

File: pylatex/table.py (hunk 1 above)
Comment (s-wallace) [inline pylatex/table.py]: That would certainly clean it up, but it doesn't appear to work:
prepending to self.data creates an environment like this:

    \begin{tabu}{X[r] X[r]}%
    to 4in%

but, what we need is:

    \begin{tabu}to 4in{X[r] X[r]}%

my reading of the ContainerCommand class and the Command class suggests that this syntax isn't supported by the assumptions of the base classes... but admittedly, I may be missing something.

**Comment by s-wallace** [comment]
I just wanted to give this a ping; do you want additional exploration before a merge?

**Pull request merged**